feconf.py

```diff
@@ -260,7 +260,7 @@
 # incompatible changes are made to the states blob schema in the data store,
 # this version number must be changed and the exploration migration job
 # executed.
-CURRENT_STATE_SCHEMA_VERSION = 47
+CURRENT_STATE_SCHEMA_VERSION = 48
```

Comment:
If we are changing the state schema version then we also need to migrate the questions?

Reply:
Yup, added now!